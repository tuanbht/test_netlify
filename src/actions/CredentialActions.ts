import { AnyAction } from 'redux';
import { SAVE_CREDENTIAL } from 'constants/ReduxActions';

const saveCredentialAction = (token: string, orderId: number): AnyAction => ({
  type: SAVE_CREDENTIAL,
  token,
  orderId,
});

export default {
  saveCredentialAction,
};
