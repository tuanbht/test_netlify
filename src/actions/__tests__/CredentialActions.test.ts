import CredentialActions from '../CredentialActions';

describe('CredentialActions', () => {
  describe('saveCredentialAction', () => {
    it('returns action save credential', () => {
      expect(CredentialActions.saveCredentialAction('token', 1)).toMatchSnapshot();
    });
  });
});
