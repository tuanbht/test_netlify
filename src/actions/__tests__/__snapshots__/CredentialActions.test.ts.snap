// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CredentialActions saveCredentialAction returns action save credential 1`] = `
Object {
  "orderId": 1,
  "token": "token",
  "type": "SAVE_CREDENTIAL",
}
`;
